components/array-object-table/src/stories.js

```diff
@@ -52,28 +53,34 @@ examples.add('skipEmtpyRows and hideWithNoValues (table is hidden if no rows)',
   ];
   const title = ['Heading'];
 
-  return <ArrayObjectTable fields={fields} array={array} title={title} skipEmptyRows hideWithNoValues/>;
+  return (<ArrayObjectTable
+    fields={fields}
+    array={array}
+    title={title}
+    skipEmptyRows
+    hideWithNoValues
+  />);
 });
 
 examples.add('With object transform and default transform', () => {
   const fields = [
     { key: 'one', heading: 'One' },
-    { key: 'two', heading: 'Two', transform: value => value ? value.toLowerCase() : '' },
+    { key: 'two', heading: 'Two', transform: value => (value ? value.toLowerCase() : '') },
     { key: 'three', heading: 'Three' },
-    { key: 'three', heading: 'Four', transform: value => value ? value.toLowerCase() : '*' },
+    { key: 'three', heading: 'Four', transform: value => (value ? value.toLowerCase() : '*') },
   ];
   const array = [
-    {one: 'One', two: 'Two'},
+    { one: 'One', two: 'Two' },
   ];
   const title = ['Heading'];
   const defaultTransform = value => (value || '-');
 
-  return <ArrayObjectTable
+  return (<ArrayObjectTable
```

Comment:
the parens here really needed?

i get the need for a bracket when there's nothing else on the line, but as this line also has the start of a JSX tag surely that's OK?

Reply:
On current config `react/jsx-wrap-multilines` objects without these - I'll take a look at options...

Reply:
ok cool - i think i'm not a fan of that rule then :grin:

Reply:
OK... There is no option enforcing that JSX is _never_ wrapped in parens but I could turn off the rule requiring that multiline JSX is wrapped.

Wrapping is not necessary for multi-line JSX, but is advised by FB and Airbnb, FB saying it "avoids the pitfalls of automatic semicolon insertion". Shall we say that we will turn off the rule, though? This will mean there _can_ be parens around both single and multiline JSX, but it is not required or "fixed" to be so. And our policy could be not to do it at all. That's fine with me.

btw If there is a `return` statement then a line break before the JSX, then yes, a bracket is needed or the return is deemed illegal, but I guess we could generally avoid this too and always begin with our JSX or opening element.